plugins/rollbar_dashboards/app/views/rollbar_dashboards/_deploy.html.erb

```diff
@@ -0,0 +1,29 @@
+<%
+  occurrence_term = 'occurrences_count'
+  if deploy.succeeded?
+%>
+  <% if deploy.project.rollbar_dashboards_settings.empty? %>
+    <%= render partial: 'rollbar_dashboards/no_dashboards', locals: { project: deploy.project } %>
+  <%
+    else
+      @data = deploy.project.rollbar_dashboards_settings.map do |dashboard|
+        params = {
+          date_from: deploy.created_at.in_time_zone(dashboard.time_zone).strftime('%m-%d-%Y %H:%M:%S'),
+          date_to: deploy.next_successful_deploy&.created_at&.in_time_zone(dashboard.time_zone)&.strftime('%m-%d-%Y %H:%M:%S') || '',
+        }
+
+        items = RollbarDashboards::Client.new(dashboard).items(params)
+        [dashboard, items&.sort_by { |i| i[occurrence_term] }&.reverse&.first(SamsonRollbarDashboards::Engine::NUM_ITEMS)]
+      end
+  %>
+    <%=
+      render partial: 'rollbar_dashboards/dashboard',
+        collection: @data,
+        as: :dashboard_data,
+        locals: {
+          title: "Top #{SamsonRollbarDashboards::Engine::NUM_ITEMS} Items That Occurred This Deploy",
```

Comment:
items -> errors ?

"Top 5 Errors During This Deploy"

Reply:
Rollbar uses the term 'Item' in it's dashboard. Do we want to copy their terminology or would 'Error' be more intuitive?